moveit_planners/pilz_industrial_motion_planner/CMakeLists.txt

```diff
@@ -8,7 +8,6 @@ add_definitions(-Wextra)
 add_definitions(-Wno-unused-parameter)
 
 find_package(catkin REQUIRED COMPONENTS
-  eigen_conversions
   kdl_conversions
```

Comment:
What about `kdl_conversions`? Is this still needed, now that you use `eigen_kdl`?